pkgs/top-level/all-packages.nix

```diff
@@ -6552,9 +6554,6 @@ in
   yaft = callPackage ../applications/misc/yaft { };
 
   yarn = callPackage ../development/tools/yarn  { };
-  
-  yarn2nix = throw "Use upstream https://github.com/moretea/yarn2nix";
-  mkYarnPackage = yarn2nix;
```

Comment:
?

Reply:
Previous commit gives me a conflict error. I remove those lines and conflict gone. is it right?

Reply:
I'm not sure what gives you an error. I will assume that you get some merge conflict because your original commit contains spurious whitespace changes that might have clashed with https://github.com/NixOS/nixpkgs/commit/32fc68db570144b9b4e05ee819067c412fa8a17d#diff-036410e9211b4336186fc613f7200b12.

There are many ways to fix this. I guess you could remake your commit from scratch:
```
git reset --soft HEAD
git reset
git add pkgs/data/fonts/nahid-fonts/default.nix
git add -p pkgs/top-level/all-packages.nix
# answer no to all hunks except one with "nahid-fonts = callPackage ../data/fonts/nahid-fonts { };"
git commit -m "nahid-fonts: init at v0.3.0"
git push -f
```

Reply:
There is no result when i run those commands.